source/_integrations/ecowitt.markdown

```diff
@@ -24,15 +24,20 @@ ha_integration_type: integration
 
 The **Ecowitt** {% term integration %} works by first creating a callback endpoint on your Home Assistant instance and then adding this configuration to the Ecowitt console so that it starts sending data.
 
-To set up this integration:
+The integration will display a Server IP / Host Name, Path, and Port. You can input it into the Ecowitt configuration in one of two ways:
 
-1. Add the Ecowitt integration to your Home Assistant instance. When doing so, the config flow will display a Server IP, Path, and Port.
-2. Use the Ecowitt App (on your phone) or access the Ecowitt WebUI in a browser at the station IP address.
-3. Select your station > Menu **Others** > **DIY Upload Servers**.
-4. Select **Next**, then select **Customized**.
-5. Select the protocol **Ecowitt**.
-6. Enter the Server IP, Path, and Port that were displayed in step 1.
-    - The path has to match! If using the Ecowitt app, remove the first forward slash from the path, as the app will prepend one.
-7. Save the configuration.
+1. Use the Ecowitt App (on your phone):
+    - Tap the ☰ on the upper left, then 'My Devices' → Pick your station
```

Comment:
Items that should be selected in the user interface must be emphasised in bold (as the original docs did).

Reply:
Use "Select" instead of "Tap" in your wording, which is a more inclusive and generic term to use for these things.